addons/l10n_dk_oioubl/models/account_move.py

```diff
@@ -0,0 +1,12 @@
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    @api.model
+    def _get_ubl_cii_builder_from_xml_tree(self, tree):
+        customization_id = tree.find('{*}CustomizationID')
+        if customization_id is not None and 'OIOUBL-2' in customization_id.text:
+            return self.env['account.edi.xml.oioubl_201']
+        super()._get_ubl_cii_builder_from_xml_tree(tree)
```

Comment:
You are breaking the super call! :angry: 
```suggestion
    def _get_ubl_cii_builder_from_xml_tree(self, tree):
        builder = super()._get_ubl_cii_builder_from_xml_tree(tree)
        if builder:
            return builder
        customization_id = tree.find('{*}CustomizationID')
        if customization_id is not None and 'OIOUBL-2' in customization_id.text:
            return self.env['account.edi.xml.oioubl_201']
```

Reply:
Why is it an issue?
It only breaks it if and only if it's an OIOUBL format.

Reply:
@Levizar you missed to return the super results

Reply:
@Levizar You need at least to do:
```suggestion
    def _get_ubl_cii_builder_from_xml_tree(self, tree):
        customization_id = tree.find('{*}CustomizationID')
        if customization_id is not None and 'OIOUBL-2' in customization_id.text:
            return self.env['account.edi.xml.oioubl_201']
        return super()._get_ubl_cii_builder_from_xml_tree(tree)
```

Reply:
You are right for that! I haven't seen it. Thanks!

But I'm a bit worried: this bug didn't break any test. :grimacing: